src/core/GuiApplication.cpp

```diff
@@ -25,7 +25,7 @@ void GuiApplication::BeginFrame()
 	m_renderer->SetRenderTarget(m_renderTarget);
 #endif
 	// TODO: render target size
-	m_renderer->SetViewport(0, 0, Graphics::GetScreenWidth(), Graphics::GetScreenHeight());
+	m_renderer->SetViewport({ 0, 0, Graphics::GetScreenWidth(), Graphics::GetScreenHeight() });
```

Comment:
I need to test this kind of initialisation with Visual Studio builds, and non GCC builds basically.

Reply:
This *should* be standard-compliant list-initialization, but don't quote me on that. Appveyor didn't build because there's a merge conflict but I don't think it had problems with it.

Returning a list-initialized object (and this particular call) is a form of `copy-list-initialization`, which as far as I can tell is directly standard-compliant according to [the C++ reference](https://en.cppreference.com/w/cpp/language/list_initialization).

Reply:
Yeah this isn't your fault or a C++ standards issue 😉 it's a does-VS-conform-to-the-C++-standards issue, which with this particular list initialisation I have seen it fail before, so I want to double check it before it's merged